batchflow/research/profiler.py

```diff
@@ -44,8 +47,8 @@ def show_profile_info(self, per_iter=False, per_experiment=False, detailed=False
         detailed = False if not self.detailed else detailed
 
         if per_iter is False and detailed is False:
-            groupby = ['experiment', 'action', 'iter'] if per_experiment else ['action', 'iter']
-            columns = columns or ['total_time', 'eval_time']
+            groupby = ['experiment', 'unit', 'iter'] if per_experiment else ['unit', 'iter']
```

Comment:
did you mean self.UNIT ?

Reply:
`self.UNIT_NAME` is for `_add_profile_info` which is common for all profilers.
We define `show_profile_info` for pipelines and experiments separately so we can write here `'unit'` directly.